src/components/Tables/Table.tsx

```diff
@@ -132,6 +133,22 @@ export type EditableConfig<Data> =
       }
     | { editable: false };
 
+export type BreakPointConfig = {
+    name: string;
+    breakPoints: number[];
+    dividingLineStyle: keyof DividingLineStyleOptions;
+};
+
+export type DividingLineStyleOptions = {
+    dateAndSlotPrimary: DividingLineStyle;
+    dateAndSlotSecondary: DividingLineStyle;
+    slotPrimary: DividingLineStyle;
+};
+
+type DividingLineStyle = {
+    colour: string;
+    thickness: string;
+};
```

Comment:
Not sure if this should also go to `conditionalStyling.ts` - see what works and looks better

Reply:
done